tests/test_chiral_restraints.py

```diff
@@ -0,0 +1,526 @@
+import multiprocessing
+import os
+
+os.environ["XLA_FLAGS"] = "--xla_force_host_platform_device_count=" + str(multiprocessing.cpu_count())
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+import scipy
+from rdkit import Chem
+
+from timemachine.fe import chiral_utils, topology, utils
+from timemachine.ff import Forcefield
+from timemachine.integrator import simulate
+from timemachine.potentials.chiral_restraints import (
+    U_chiral_atom,
+    U_chiral_atom_batch,
+    U_chiral_bond,
+    U_chiral_bond_batch,
+    pyramidal_volume,
+    torsion_volume,
+)
+
+
+def minimize_scipy(x0, U_fn):
+    N = x0.shape[0]
+
+    def U_bfgs(x_flat):
```

Comment:
Naming: `U_bfgs` -> `U_flat`?

(The energy function isn't modified by the optimize.)

Reply:
willfix

Reply:
willfix